app/timetables/models.py

```diff
@@ -175,3 +175,16 @@ def __str__(self):
 
     class Meta:
         unique_together = ('timetable', 'cycle_day', 'meal', 'meal_option')
+
+
+class Vendor(SlugifyMixin, TimestampMixin):
+    """Model representing food service-provider"""
+
+    name = models.CharField(max_length=255)
+    slug = models.SlugField(max_length=255, unique=True, editable=False)
```

Comment:
@andela-kerinoso That makes sense. Thanks for clarifying. If the `slug` field is non-editable and used for enforcing uniqueness internally, wouldn't it make sense to remove it from the admin altogether?